data_analysis/model_ad/notebooks/MG-402_Create ui_config source file.rmd

```diff
@@ -334,49 +409,26 @@ mo_filter_values <- list(
 
 # Generates Model Overview ui_config objects
 build_mo_object <- function() {
-  columns <- apply(mo_columns, 1, function(row) {
-    name <- row["name"]
-    type <- row["type"]
-
-    metadata <- c(type, "", paste("Sort by", name, "value"))
-    column <- make_column(name, metadata)
-
-    if (column$type %in% c("link_internal", "link_external")) {
-      if (name %in% c("Gene Expression", "Disease Correlation", "Pathology", "Biomarkers")) {
-        column$link_text <- "Results"
-      } else if (name == "Study Data") {
-        column$link_text <- "View Data"
-      } else if (name == "JAX Strain") {
-        column$link_text <- "View Strain"
-      } else if (name == "Center") {
-        # Omit link_text, add link_url instead
-        column$link_url <- "https://www.model-ad.org/"
```

Comment:
I don't see this link put anywhere else and the link_url variable above is all NAs, is this intentional?